src/Monolog/Handler/TelegramBotHandler.php

```diff
@@ -122,12 +122,31 @@ public function disableNotification(bool $disableNotification = null): self
         return $this;
     }
 
+    /**
+     * {@inheritdoc}
+     */
+    public function handleBatch(array $records): void
+    {
+        $messages = [];
+
+        foreach ($records as $record) {
+            if ($record['level'] < $this->level) {
+                continue;
+            }
+            $messages[] = $this->processRecord($record);
+        }
+
+        if (!empty($messages)) {
+            $this->send((string) $this->getFormatter()->formatBatch($messages), $messages);
+        }
+    }
+
     /**
      * @inheritDoc
      */
     protected function write(array $record): void
     {
-        $this->send($record['formatted']);
+        $this->send($record['formatted'], [$record['formatted']]);
```

Comment:
I don't see how this fixes anything? The send method receives only one argument.

Reply:
@Seldaek Yeah. I missed it. I've added some corrections to the next commit.

Reply:
I still don't get the point, $records isn't used in the send method, so why is this passed at all?

Reply:
Also handleBatch calls send with a single argument, so it's broken now that you added the second required argument.